scripts/setup.py

```diff
@@ -213,6 +213,9 @@ def main(args=None):
     elif os.path.isfile('/usr/bin/chromium-browser'):
         # Unix.
         chrome_bin = '/usr/bin/chromium-browser'
+    elif os.path.isfile('/usr/bin/brave'):
+        # Arch Linux.
+        chrome_bin = '/usr/bin/brave'
```

Comment:
Can't we install chrome/chromium in Arch Linux? (Asking this as the above code-comment sounds like brave is the only option in Arch (not sure!)) Also, can you confirm whether webdriver works fine with brave (As we use this file/function while running e2e test))?

Reply:
@DubeySandeep brave is constantly gaining popularity in arch distributions for sometime . So most of the users prefer brave as their main go-to browser over chromium . I thought adding brave support will provide out-of-the-box compatiblity for many users , as they are using brave . For the comment you mentioned , i think it will work fine with any other linux distribution , as long as the brave script is installed in the right directory ( We can certainly change that comment ) . For the last question , i have tried doing the e2e tests with this changes , and they worked fine without any problem . Same was the case with other tests , they worked all fine . I cannot fully guarantee the compatablity of these changes , but so far i have not encountered any problem working with these changes .

Reply:
@ashish-patwal I want to make sure I get these 2 points clarified before moving ahead:

1. Does Brave Officially support Arch Linux?
2. Is there a statistic showing that brave started gaining a good market share of users.

Reply:
1. Yes brave is official supported by arch . It is published on main arch repos not the AUR ( which is kind of more community oriented )
2. I can't prove that fact . But most of the time on reddit , community polls , gitter chats , slack etc. i often see people going for brave as their daily browser . It is also due to the fact that brave gives many interesting features . Some of my favourites include tor integration , transparency towards user privacy , top knotch ad-blocking , firewall + VPN integration , chrome extension/plugin support .

Reply:
AUR is not supported by Arch right?

Reply:
@DubeySandeep Do we have to test for Brave during release testing?

Reply:
> AUR helpers are not supported by Arch right? Just trying to make sure in the future developers don't face any problems.

Yes they are not supported by arch as AUR is a community driven repository .